lib/AFS/CellCC/Util.pm

```diff
@@ -331,4 +332,58 @@ scratch_ok($$$$$) {
     return 1;
 }
 
+# Return the checksum of the file received as an argument.
+# The checksum returned by this function will be in following
+# format: "algorithm:checksum", where algorithm is the algorithm
+# of choice (e.g. MD5) and checksum is the checksum of the file
+# received as an argument. This checksum will be in hexadecimal form.
+sub
+calc_checksum($$$$$$) {
+    my ($dumpfh, $filesize, $algo, $jobid, $dvref, $state) = @_;
+
+    if ($filesize < 0) {
+        die("The size of the file cannot be negative\n");
+    }
+
+    my $start = time();
+    my $now = $start;
+
+    my $total = pretty_bytes($filesize);
+    my $bytes;
+    my $nbytes;
+
+    my $digest = Digest->new($algo);
```

Comment:
Since this module is now using the Digest module, add a 'use Digest;' near the beginning of this file. Currently I believe this doesn't cause an error, because the only other places in the code (Dump.pm and Restore.pm) of course already 'use Digest'. Also remove the 'use Digest' from Dump.pm (but not Restore.pm, since Restore.pm still uses Digest directly).